assets/controller.yaml

```diff
@@ -136,5 +148,16 @@ spec:
               memory: 50Mi
               cpu: 10m
       volumes:
+        - name: aws-credentials
+          secret:
+            secretName: ebs-cloud-credentials
+        # This service account token can be used to provide identity outside the cluster.
+        # For example, this token can be used with AssumeRoleWithWebIdentity to authenticate with AWS using IAM OIDC provider and STS.
+        - name: bound-sa-token
+          projected:
+            sources:
+            - serviceAccountToken:
+                path: token
+                audience: openshift
```

Comment:
Did we wanted to set `expirationSeconds` here?

Reply:
There is no requirement currenly to have a specific duration, so i decided to fallback to default for now.